### ISSUE TITLE
Don't raise SC2086 if IFS appears to be empty.

### ISSUE THREAD

**Issue by Arthur2e5** [koalaman/shellcheck#511]
Consider this script:
```Bash
IFS='' # or IFS=
echo $BASH_VERSION
```

**Comment by knirch** [comment]
Globbing is still an issue regardless if IFS is empty or not.
